model/lifecycle.go

```diff
@@ -1563,7 +1562,7 @@ func addNewBuilds(ctx context.Context, creationInfo TaskCreationInfo, existingBu
 		grip.Info(message.Fields{
 			"op":        "creating build for version",
 			"variant":   pair.Variant,
-			"activated": activateVariant,
+			"activated": true,
```

Comment:
Do we need to log this anymore if it's always true? Seems like this (+ the one below) could just be removed since it's implied.

Reply:
Sounds good, removing